src/common/process/test/ProcessTest.cpp

```diff
@@ -43,12 +43,15 @@ TEST(ProcessUtils, isPidAvailable) {
     }
     {
         // pid file which contains pid of current process
-        auto pidFile = "/tmp/maybe-non-existing-dir/process_test.pid";
+        auto pidFile = folly::stringPrintf("/tmp/non-existing-dir-%d/process_test.pid", ::getpid());
```

Comment:
Why not TempDir?

Reply:
I had considered it.

But TempDir for TempFile will create the directory or file and be capable of deleting them automatically. But what I need here is a temp name and auto deletion, but not the file creation.

Reply:
Fair enough